.storybook/preview.js

```diff
@@ -0,0 +1,16 @@
+// Order is important!
```

Comment:
nit: add comment that this must match pages/_app.tsx (unfortunately next requires the imports directly in that file, otherwise we could share)